pipelines/tekton/resources.go

```diff
@@ -230,7 +236,13 @@ func getBuilderImage(f fn.Function) (name string) {
 }
 
 func getPipelineName(f fn.Function) string {
-	return fmt.Sprintf("%s-%s-pipeline", f.Name, f.Build.Builder)
+	var source string
+	if f.Build.Git.URL == "" {
+		source = "direct-upload"
```

Comment:
```suggestion
		source = "upload"
```

Just a nit to make the name shorter